src/main/groovy/org/savantbuild/plugin/dep/idea/IDEAPlugin.groovy

```diff
@@ -139,12 +139,21 @@ class IDEAPlugin extends BaseGroovyPlugin {
 
   private toRelativePATH(Path path, String userHome) {
     def artifactRealPath = path.toRealPath().toString()
+    // If we have a cache process defined, getting the first one is useful because it gives a more precise
```

Comment:
this was not part of the fix directly, but it did enable me to run the tests with different paths and not fail (and would also be useful in a GHA runner for the same purpose)